include/test/helper/fixture.hpp

```diff
@@ -85,4 +85,27 @@ create_nil_entry( void );
 const char *
 load_corpus( const std::string& name );
 
+struct stumpless_test_data {
+    int length;
+    char **test_strings;
+};
```

Comment:
Use a C++ vector or other data structure here, instead of manually managing a list of strings this way.